src/_rules/color.js

```diff
@@ -12,6 +12,7 @@ export const textColors = [
 //  ['text-inherit', { 'color': 'inherit' }], // This class currently sets "text-align: inherit;" in align.js
   ['text-transparent', { 'color': 'transparent' }],
   ['text-current', { 'color': 'currentColor' }],
+  [/^text-\[(.+)]/, ([, p]) => ({ 'color': p.startsWith('var') ? p : `var(${p})` })],
```

Comment:
Since we use `^` to match the beginning, shouldn't there be `$` after `]` to mark the end of the string?